packages/aws-lambda/layer/bin/publish-layer.sh

```diff
@@ -27,24 +27,26 @@ EOF
 }
 
 PACKAGE_NAMES="@instana/aws-lambda instana-aws-lambda-auto-wrap"
-LAYER_NAME=instana-nodejs
+LAYER_NAME=hmadison-instana-nodejs
```

Comment:
Just as a reminder for ourselves, this needs to be reverted at some point, shortly before merging this.